src/stactools/usgs_nlcd/constants.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+
+from pyproj import CRS
+from pystac import Link, Provider, ProviderRole
+
+NLCD_ID = "USGS_NLCD"
+NLCD_EPSG = 4326
+NLCD_CRS = CRS.from_epsg(NLCD_EPSG)
+LICENSE = "proprietary"
+lic_link = """"https://github.com/stactools-packages/usgs-nlcd/tree/main/src/stactools/usgs_nlcd/license
+/license.txt"""
```

Comment:
The file linked here just got deleted from the repo. What's the plan for this?
The triple quotes will include a line ending in the middle of the string which won't work for a URL.